bundles/org.openhab.ui.homebuilder/pom.xml

```diff
@@ -73,7 +73,7 @@
           <execution>
             <id>npm install</id>
             <goals>
-              <goal>npm</goal>
+              <goal>npm --no-save</goal>
```

Comment:
The goal should remain the same. Just add the configuration like I did. :wink: or it will error:

`[ERROR] Could not find goal 'npm --no-save' in plugin com.github.eirslett:frontend-maven-plugin:1.0 among available goals grunt, install-node-and-npm, jspm, bower, karma, webpack, gulp, npm, ember -> [Help `]`

Reply:
Wout was trying to help so my focus was not that high, changing now and testing locally before pushing :-)

Reply:
You can't start helping with openHAB young enough. ;-)